Failed to call observer method

```
FATAL EXCEPTION: main
Process: org.ole.planet.myplanet, PID: 24260
java.lang.RuntimeException: Failed to call observer method
	at androidx.lifecycle.ClassesInfoCache$MethodReference.invokeCallback(ClassesInfoCache.java:232)
	at androidx.lifecycle.ClassesInfoCache$CallbackInfo.invokeMethodsForEvent(ClassesInfoCache.java:199)
	at androidx.lifecycle.ClassesInfoCache$CallbackInfo.invokeCallbacks(ClassesInfoCache.java:190)
	at androidx.lifecycle.ReflectiveGenericLifecycleObserver.onStateChanged(ReflectiveGenericLifecycleObserver.java:40)
	at androidx.lifecycle.LifecycleRegistry$ObserverWithState.dispatchEvent(LifecycleRegistry.kt:314)
	at androidx.lifecycle.LifecycleRegistry.forwardPass(LifecycleRegistry.kt:251)
	at androidx.lifecycle.LifecycleRegistry.sync(LifecycleRegistry.kt:287)
	at androidx.lifecycle.LifecycleRegistry.moveToState(LifecycleRegistry.kt:136)
	at androidx.lifecycle.LifecycleRegistry.handleLifecycleEvent(LifecycleRegistry.kt:119)
	at androidx.lifecycle.ReportFragment$Companion.dispatch$lifecycle_runtime_release(ReportFragment.kt:192)
	at androidx.lifecycle.ReportFragment$LifecycleCallbacks.onActivityPostStarted(ReportFragment.kt:121)
	at android.app.Activity.dispatchActivityPostStarted(Activity.java:1466)
	at android.app.Activity.performStart(Activity.java:8667)
	at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3807)
	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:225)
	at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:205)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:177)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:98)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2443)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:205)
	at android.os.Looper.loop(Looper.java:294)
	at android.app.ActivityThread.main(ActivityThread.java:8177)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
Caused by: java.lang.NullPointerException
	at org.ole.planet.myplanet.ui.userprofile.UserProfileFragment.getBinding(UserProfileFragment.kt:70)
	at org.ole.planet.myplanet.ui.userprofile.UserProfileFragment.access$getBinding(UserProfileFragment.kt:67)
	at org.ole.planet.myplanet.ui.userprofile.UserProfileFragment$loadProfileImage$1$1.onLoadFailed(UserProfileFragment.kt:187)
	at com.bumptech.glide.request.SingleRequest.onLoadFailed(SingleRequest.java:715)
	at com.bumptech.glide.request.SingleRequest.begin(SingleRequest.java:228)
	at com.bumptech.glide.manager.RequestTracker.resumeRequests(RequestTracker.java:115)
	at com.bumptech.glide.RequestManager.resumeRequests(RequestManager.java:339)
	at com.bumptech.glide.RequestManager.onStart(RequestManager.java:364)
	at com.bumptech.glide.manager.LifecycleLifecycle.onStart(LifecycleLifecycle.java:28)
	at java.lang.reflect.Method.invoke(Native Method)
	at androidx.lifecycle.ClassesInfoCache$MethodReference.invokeCallback(ClassesInfoCache.java:225)
	at androidx.lifecycle.ClassesInfoCache$CallbackInfo.invokeMethodsForEvent(ClassesInfoCache.java:199) 
	at androidx.lifecycle.ClassesInfoCache$CallbackInfo.invokeCallbacks(ClassesInfoCache.java:190) 
	at androidx.lifecycle.ReflectiveGenericLifecycleObserver.onStateChanged(ReflectiveGenericLifecycleObserver.java:40) 
	at androidx.lifecycle.LifecycleRegistry$ObserverWithState.dispatchEvent(LifecycleRegistry.kt:314) 
	at androidx.lifecycle.LifecycleRegistry.forwardPass(LifecycleRegistry.kt:251) 
	at androidx.lifecycle.LifecycleRegistry.sync(LifecycleRegistry.kt:287) 
	at androidx.lifecycle.LifecycleRegistry.moveToState(LifecycleRegistry.kt:136) 
	at androidx.lifecycle.LifecycleRegistry.handleLifecycleEvent(LifecycleRegistry.kt:119) 
	at androidx.lifecycle.ReportFragment$Companion.dispatch$lifecycle_runtime_release(ReportFragment.kt:192) 
	at androidx.lifecycle.ReportFragment$LifecycleCallbacks.onActivityPostStarted(ReportFragment.kt:121) 
	at android.app.Activity.dispatchActivityPostStarted(Activity.java:1466) 
	at android.app.Activity.performStart(Activity.java:8667) 
	at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3807) 
	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:225) 
	at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:205) 
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:177) 
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:98) 
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2443) 
	at android.os.Handler.dispatchMessage(Handler.java:106) 
	at android.os.Looper.loopOnce(Looper.java:205) 
	at android.os.Looper.loop(Looper.java:294) 
	at android.app.ActivityThread.main(ActivityThread.java:8177) 
	at java.lang.reflect.Method.invoke(Native Method) 
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552) 
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971) 
```

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -10,8 +10,8 @@ android {
         applicationId "org.ole.planet.myplanet"
         minSdk = 26
         targetSdk = 36
-        versionCode = 3156
-        versionName = "0.31.56"
+        versionCode = 3167
+        versionName = "0.31.67"
         ndkVersion = '26.3.11579264'
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         vectorDrawables.useSupportLibrary = true

--- a/app/src/main/java/org/ole/planet/myplanet/ui/userprofile/UserProfileFragment.kt
+++ b/app/src/main/java/org/ole/planet/myplanet/ui/userprofile/UserProfileFragment.kt
@@ -195,13 +195,14 @@ class UserProfileFragment : Fragment() {
                 override fun onLoadFailed(
                     e: GlideException?,
                     model: Any?,
-                    target: Target<Drawable>?,
+                    target: Target<Drawable>,
                     isFirstResource: Boolean
                 ): Boolean {
-                    if (!isAdded || _binding == null) {
+                    if (!isAdded) {
                         return true
                     }
-                    _binding?.image?.apply {
+                    val currentBinding = _binding ?: return true
+                    currentBinding.image.apply {
                         visibility = View.VISIBLE
                         setImageResource(R.drawable.profile)
                     }

--- a/app/src/main/java/org/ole/planet/myplanet/ui/userprofile/UserProfileFragment.kt
+++ b/app/src/main/java/org/ole/planet/myplanet/ui/userprofile/UserProfileFragment.kt
@@ -178,23 +178,47 @@ class UserProfileFragment : Fragment() {
     }
 
     private fun loadProfileImage() {
-        model?.userImage.let {
-            Glide.with(requireContext())
-                .load(it)
-                .apply(RequestOptions().placeholder(R.drawable.profile).error(R.drawable.profile))
-                .listener(object : RequestListener<Drawable> {
-                    override fun onLoadFailed(e: GlideException?, model: Any?, target: Target<Drawable>, isFirstResource: Boolean): Boolean {
-                        binding.image.setImageResource(R.drawable.profile)
-                        return false
-                    }
+        val binding = _binding ?: return
+        val profileImageUrl = model?.userImage
 
-                    override fun onResourceReady(resource: Drawable, model: Any, target: Target<Drawable>?, dataSource: DataSource, isFirstResource: Boolean): Boolean {
-                        return false
-                    }
-
-                })
-                .into(binding.image)
+        if (profileImageUrl.isNullOrBlank()) {
+            binding.image.setImageResource(R.drawable.profile)
+            return
         }
+
+        if (!isAdded) return
+
+        Glide.with(this)
+            .load(profileImageUrl)
+            .apply(RequestOptions().placeholder(R.drawable.profile).error(R.drawable.profile))
+            .listener(object : RequestListener<Drawable> {
+                override fun onLoadFailed(
+                    e: GlideException?,
+                    model: Any?,
+                    target: Target<Drawable>?,
+                    isFirstResource: Boolean
+                ): Boolean {
+                    if (!isAdded || _binding == null) {
+                        return true
+                    }
+                    _binding?.image?.apply {
+                        visibility = View.VISIBLE
+                        setImageResource(R.drawable.profile)
+                    }
+                    return true
+                }
+
+                override fun onResourceReady(
+                    resource: Drawable,
+                    model: Any,
+                    target: Target<Drawable>?,
+                    dataSource: DataSource,
+                    isFirstResource: Boolean
+                ): Boolean {
+                    return false
+                }
+            })
+            .into(binding.image)
     }
 
     private fun openEditProfileDialog() {

